cache/in-memory/src/event/guild.rs

```diff
@@ -455,4 +457,72 @@ mod tests {
         assert_eq!(cache.guild(guild.id).unwrap().owner_id, mutation.owner_id);
         assert_eq!(cache.guild(guild.id).unwrap().id, mutation.id);
     }
+
+    #[test]
+    fn test_guild_member_count() {
+        let user_id = Id::new(2);
+        let guild_id = Id::new(1);
+        let cache = InMemoryCache::new();
+        let user = test::user(user_id);
+        let member = test::member(user_id, guild_id);
+
+        let guild = Guild {
```

Comment:
is it possible to use the guild scaffolding in `crate::test`?

Reply:
There was none previously so I added one now